portfolio/src/main/webapp/comment-page/comment.js

```diff
@@ -170,4 +175,8 @@ function fetchLogout() {
       location.replace(status.url); //go to logout page
     } 
   });
+}
+
+function translate() {
+
 }
```

Comment:
I too like empty functions! Is this one needed?

Reply:
That was an accident - I removed it!